pyproject.toml

```diff
@@ -44,6 +44,7 @@ requires-python = ">=3.7"
 [project.optional-dependencies]
 test = [
     "coverage[toml]",
+    "poetry",
```

Comment:
xref the comment about depending on `poetry` -- I don't think we want this dep, even as a test-only dep.

Reply:
Alrighty, I'll drop it for now. But I suggest having integration tests for poetry at some point. When I was doing [dephell](https://github.com/dephell/dephell/), I remember poetry support breaking quite often, mostly because of new features that users expected dephell to support too (like a list of constraints for a single dependency). But poetry should be more stable now, I think.

Reply:
Rewrote tests, removed poetry from dependencies ✅